pkg/build/go.mod

```diff
@@ -7,11 +7,6 @@ go 1.23.1
 // github.com/docker/docker@v0.0.0-00010101000000-000000000000: invalid version: unknown revision 000000000000
 replace github.com/docker/docker => github.com/moby/moby v26.0.0+incompatible
 
-// Override Prometheus version because Prometheus v2.X is tagged as v0.X for Go modules purposes and Go assumes
-// that v1.Y is higher than v0.X, so when we resolve dependencies if any dependency imports v1.Y we'd
-// import that instead of v0.X even though v0.X is newer.
-replace github.com/prometheus/prometheus => github.com/prometheus/prometheus v0.52.0
-
```

Comment:
This was removed but accidently brought back by https://github.com/grafana/grafana/pull/98742

Reply:
It was removed from root `go.mod` https://github.com/grafana/grafana/pull/96925 and after that PR again herehttps://github.com/grafana/grafana/pull/98848